xwiki-platform-core/xwiki-platform-bridge/src/main/java/org/xwiki/bridge/event/DocumentMovedEvent.java

```diff
@@ -44,25 +44,42 @@
      */
     private static final long serialVersionUID = 1L;
     
-    /**
-     * Constructor initializing the event filter with an
-     * {@link org.xwiki.observation.event.filter.AlwaysMatchingEventFilter}, meaning that this event will match any other document delete event.
+   /**
+     * The move document is about to be moved to.
+     */
+    private String move;
+   
+   /**
+     * Matches all {@link DocumentMovedEvent} events.
      */
     public DocumentMovedEvent()
     {
         super();
     }
-    
+
+    /**
+     * Matches {@link DocumentMovedEvent} events that target the specified document.
+     * 
+     * @param documentReference the reference of the document to match
+     */
+    public DocumentMovedEvent(DocumentReference documentReference)
+    {
+        this(documentReference, null);
+    }
+
     /**
-     * Constructor initializing the event filter with a {@link org.xwiki.observation.event.filter.FixedNameEventFilter}, meaning that this event will match only delete events affecting the same document.
+     * Matches {@link DocumentMovedEvent} events that target the specified document and move. The move is
+     * matched only if it's not {@code null}.
      * 
      * @param documentReference the reference of the document to match
+     * @param move the move the document was moved to
      */
-    public DocumentMovedEvent(DocumentReference documentReference)    
+    public DocumentMovedEvent(DocumentReference documentReference, String move)
```

Comment:
I think the moved document should be contained in the data part instead.

Reply:
But this also works fine for me